src/components/card/card.ts

```diff
@@ -47,11 +64,22 @@ export class Card extends LitElement {
           href=${this.href}
           target=${this.target}
           rel=${this.rel}
+          role=${ifDefined(this.cardRole)}
+          aria-label="${ifDefined(
+            this.cardRole ? this.cardDescription : undefined
+          )}"
           @click=${(e: Event) => this.handleClick(e)}
         >
           <slot></slot>
         </a>`
-      : html`<div part="card-wrapper" class="card-wrapper">
+      : html`<div
+          part="card-wrapper"
+          class="card-wrapper"
+          role="${ifDefined(this.cardRole)}"
+          aria-label=${ifDefined(
+            this.cardRole ? this.cardDescription : undefined
+          )}
```

Comment:
You can remove these attributes from the non-clickable variant.

Reply:
Added them as there is an accessibility violation for notification which says all the notifications should have ARIA role as `alertdialog`.

Reply:
After reading up a bit on roles for notifications, I don't think alertdialog is the correct choice: https://carbondesignsystem.com/components/notification/accessibility/